ms/segrenam.pl

```diff
@@ -7,17 +7,17 @@
 # first argument can specify custom suffix...
 $suffix=(@ARGV[0]=~/^\$/) ? shift(@ARGV) : "\$m";
 #################################################################
-# rename segments in COFF modules according to %map table below	#
-%map=(	".text" => "fipstx$suffix",				#
-	".text\$"=> "fipstx$suffix",				#
-	".rdata"=> "fipsrd$suffix",				#
-	".data" => "fipsda$suffix"	);			#
+# rename segments in COFF modules according to %map table below #
+%map=(  ".text" => "fipstx$suffix",                             #
```

Comment:
... Ah!  Because that's what the tab expanded to, ok.  I'd reduce that to one space.